app/views/hyrax/dashboard/collections/_show_subcollection_actions.html.erb

```diff
@@ -1,26 +1,30 @@
 <% if presenter.collection_type_is_nestable? %>
   <!-- The user should have deposit access to the parent (the collection we are showing) and read access to the child -->
   <% if presenter.user_can_nest_collection? %>
-    <div class="actions-controls-collections">
-      <div class="sr-only"><% t('hyrax.collection.actions.nest_collections.desc') %></div>
-      <%= button_tag '',
-                    class: 'btn btn-default pull-right add-subcollection',
-                    onclick: "$('#add-subcollection-modal-#{presenter.id}').modal('show');",
-                    title: t('hyrax.collection.actions.nest_collections.desc'),
-                    type: 'button',
-                    data: { nestable: true,
-                            hasaccess: true } do %>
-                    <%= t('hyrax.collection.actions.nest_collections.button_label') %>
-                  <% end %>
-  <!-- The user should must have the ability to create a new collection of parent's type -->
-      <% if presenter.user_can_create_new_nest_collection? %>
-        <div class="sr-only"><% t('hyrax.collection.actions.add_new_nested_collection.desc') %></div>
-        <%= link_to t('hyrax.collection.actions.add_new_nested_collection.label'),
-          hyrax.dashboard_create_subcollection_under_path(parent_id: presenter.id),
-          title: t('hyrax.collection.actions.add_new_nested_collection.desc'),
-          class: 'btn btn-default pull-right',
-          data: { turbolinks: false } %>
-      <% end %>
+    <div class="text-right">
+      <div>
+        <div class="sr-only"><% t('hyrax.collection.actions.nest_collections.desc') %></div>
+        <%= button_tag '',
+              class: 'btn btn-primary add-subcollection',
+              onclick: "$('#add-subcollection-modal-#{presenter.id}').modal('show');",
+              title: t('hyrax.collection.actions.nest_collections.desc'),
+              type: 'button',
```

Comment:
Necessary?

Reply:
This is the button option to nest an existing collection as a child of the collection. It is consistent with the ability to nest an existing work on the collection.

Reply:
This is the option to add an existing collection as a nested collection (I believe). The placement of these options are consistent with the corresponding options available for adding works to the collection.

Reply:
I removed `type: 'button'` and it displays still.  Maybe just redundant.